APITesters/ObjCAPITester/ObjCAPITester/RCPurchasesErrorCodeAPI.m

```diff
@@ -46,9 +46,7 @@ + (void)checkEnums {
         case RCInvalidSubscriberAttributesError:
         case RCLogOutAnonymousUserError:
         case RCEmptySubscriberAttributesError:
-        case RCProductDiscountMissingIdentifierError:
```

Comment:
These are still removed here 😬

Reply:
Oooops, thanks for catching it!

Reply:
https://github.com/RevenueCat/purchases-ios/pull/1173